internal/pkg/diff/diff_test.go

```diff
@@ -763,6 +763,8 @@ func createProjectState(t *testing.T) *state.State {
 	options := state.NewOptions(m, storageApi, schedulerApi, context.Background(), logger)
 	options.LoadLocalState = false
 	options.LoadRemoteState = false
-	s, _ := state.LoadState(options)
+	s, _, localErr, remoteErr := state.LoadState(options)
```

Comment:
`LoadState` nereturnovalo chyby, ale boli na to separatne metody `LocalErrors, RemoteErrors`.
To v minulosti malo zmysel, ale teraz uz nema a v tomto PR to upravujem aby sa chyby spracovavali standardnym sposobom.